### PR TITLE
Manage multiple memcached instances

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -6,25 +6,35 @@
       $package_name      = 'memcached'
       $package_provider  = undef
       $service_name      = 'memcached'
+      $service_path      = '/lib/systemd/system/'
```
Comment (saz) [inline manifests/params.pp:9]: Is there a reason, why you're using /lib here? /etc seems better fitting for me (as it's not managed by a package) but maybe I'm missing something here?

Comment (bastelfreak) [reply manifests/params.pp]: ups. I didn't do the rebase properly. fixed it

**Comment by saz** [comment]
Pull request looks good, except for one thing:
This will introduce a hard dependency on systemd for separate instances, but this module supports FreeBSD and Windows.
Maybe it would be good to have a check and fail on those systems with a proper error message?

**Comment by bastelfreak** [comment]
@saz updated it

**Comment by saz** [comment]
Looks good to me! As soon as travis is finished, I'll merge this PR

**Comment by saz** [comment]
@bastelfreak Can you have a look at travis?

**Comment by bastelfreak** [comment]
oh this shitter linter :( I fixed the warning. let's just hope that travis finishes *today*....

**Comment by saz** [comment]
Yeah... I'm keeping an eye on it

**Pull request merged**